### PR TITLE
generate docs from docstrings

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
@hodgestar ready for review

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@ results
 
 npm-debug.log
 node_modules
+
+_build
```
Comment (hodgestar) [inline .gitignore]: GitHub says there is a missing newline here.

File: requirements-docs.pip [hunk 2]
```diff
@@ -0,0 +1 @@
+sphinxcontrib-autojs==0.1dev-20110917
```
Comment (hodgestar) [inline requirements-docs.pip]: Also a missing newline here.

**Comment by hodgestar** [comment]
We don't have a version bumping script. Maybe we should have one now that the version is written in two places (package.json and docs/config.py)?

**Comment by hodgestar** [comment]
Otherwise looks good.

**Comment by rudigiesler** [comment]
I'll just get a version script from another similar project.

**Comment by rudigiesler** [comment]
@hodgestar version script added

**Comment by hodgestar** [comment]
Woot. :+1:

**Pull request merged**